dumper/app/build.gradle

```diff
@@ -70,6 +70,8 @@ dependencies {
     runtimeOnly "org.postgresql:postgresql"
     runtimeOnly "net.snowflake:snowflake-jdbc"
     runtimeOnly "com.amazon.redshift:redshift-jdbc42"
+    runtimeOnly "com.amazonaws:aws-java-sdk-core:1.12.521"
+    runtimeOnly "com.amazonaws:aws-java-sdk-redshift:1.12.520"
```

Comment:
1. Do we need both of those, or does sdk-redshift bring in sdk-core transitively?
2. Can we specify the version in the conventions file, so the version is used consistently, globally, and refer only to the artefact name here?

Reply:
1. Core can be removed as sdk-redshift depends on it. I removed it.
2. Sure. Done.